permissions/plugin-jwt-support.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "jwt-support"
+github: "jenkinsci/jwt-support-plugin"
+paths:
+- "org/jenkins-ci/plugins/jwt-support"
```

Comment:
Note: for newer plugins, there's a trend to use `io.jenkins.plugins` instead. You _might_ want to consider changing this before the first release (?).

Reply:
Good to know - thanks Batmat

Reply:
Let me change this quickly and I'll re-submit in a few minutes.

Reply:
I've changed this now - @batmat can you take a second look?

Reply:
For reference, adapted in https://github.com/jenkinsci/jwt-support-plugin/pull/5